content/300-guides/050-database/100-developing-with-prisma-migrate/050-add-prisma-migrate-to-a-project.mdx

```diff
@@ -6,14 +6,14 @@ tocDepth: 2
 
 <TopBlock>
 
-<Admonition type="warning">
+This guide describes how to add [Prisma Migrate](/concepts/components/prisma-migrate) to an existing project.
 
-**MongoDB not supported** <br />
-Prisma Migrate does not currently support the [MongoDB connector](/concepts/database-connectors/mongodb).
+<Admonition type="info">
 
-</Admonition>
+This guide **does not apply for MongoDB**.<br />
+Instead of `migrate dev`, [`db push`](/concepts/components/prisma-migrate/db-push) is used for [MongoDB](/concepts/database-connectors/mongodb).
 
-This guide describes how to add [Prisma Migrate](/concepts/components/prisma-migrate) to an existing project.
+</Admonition>
```

Comment:
Point out that Migrate for MongoDB is `db push` instead of "not supported" warning.